src/test/java/IMDbBirthdays.java

```diff
@@ -0,0 +1,40 @@
+import com.codeborne.selenide.ElementsCollection;
+import com.codeborne.selenide.SelenideElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import static com.codeborne.selenide.Selenide.*;
+
+public class IMDbBirthdays {
+
+    @Test
+    public void verifyCelebritiesWithTodayBirthday() {
+        open("https://www.imdb.com/feature/bornondate/?ref_=nv_cel_brn");
+
+        SelenideElement bornTodayElement = $x("//span[text()='Born Today']");
```

Comment:
As you're opening already the intended page open("https://www.imdb.com/feature/bornondate/?ref_=nv_cel_brn");, then you're not using bornTodayElement, let's delete unused code, we don't want visual noise in the code.